packages/repo-tools/src/generate-configs.mts

```diff
@@ -72,9 +72,8 @@ async function main(): Promise<void> {
   }
 
   const RULE_NAME_PREFIX = '@typescript-eslint/';
-  const MAX_RULE_NAME_LENGTH = Object.keys(eslintPlugin.rules).reduce(
-    (acc, name) => Math.max(acc, name.length),
-    0,
+  const MAX_RULE_NAME_LENGTH = Math.max(
+    ...Object.keys(eslintPlugin.rules).map(name => name.length),
```

Comment:
Won't affect anything since this constant is only given to `String.padEnd`, and, from MDN,
> If the value is less than or equal to str.length, the current string will be returned as-is.

I wouldn't want to add `0,`, because it could seem like the `0,` is required, to a future reader, when it actually won't affect anything.